CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ Mapbox welcomes participation and contributions from everyone. Please read [`Con
  - Fixed a crash when calling the `Style.removeImage` method with the name of a nonexistent image. ([#16391](https://github.com/mapbox/mapbox-gl-native/pull/16391))
  - Fixed a crash when encountering an invalid polyline. ([#16409](https://github.com/mapbox/mapbox-gl-native/pull/16409))
  - Certain logging statements no longer run on the main thread. ([#16325](https://github.com/mapbox/mapbox-gl-native/pull/16325))
+ - Fixed library load failure on x86 devices. ([#368](https://github.com/mapbox/mapbox-gl-native-android/pull/368))
```

Comment:
```suggestion
 - Worked around a crash that could be triggered by SoLoader. ([#368](https://github.com/mapbox/mapbox-gl-native-android/pull/368))
```

Reply:
Does https://github.com/mapbox/mapbox-gl-native-android/pull/368/ only affect x86 devices? I didn't see that in the PR

Reply:
I'm also thinking these two new entries fit better in a new sub-section "Other changes", what do you think?

### Other changes

Reply:
Yeah, only affect x86 devices.
Other changes sounds good.